### PR TITLE
Add component doc verificaion script

### PR DESCRIPTION
**Pull request by asuresh4**
Depends on https://github.com/open-telemetry/opentelemetry-collector/pull/1090

**Description:** Add script to verify component documentation exists.

**Link to tracking Issue:** https://github.com/open-telemetry/opentelemetry-collector/issues/1025

Will open issues for missing READMEs (This PR adds empty README.md files so the test will pass).

### CODE REVIEW

**Comment by asuresh4** [comment]
FYI @jrcamp

File: exporter/stackdriverexporter/README.md [hunk 1]
```diff
@@ -0,0 +1,3 @@
+# Stackdriver Exporter
```
Comment (jrcamp) [inline exporter/stackdriverexporter/README.md:1]: Maybe we should open an issue for each of these and assign them to the various vendors/owners?

**Comment by jrcamp** [comment]
lgtm

File: exporter/stackdriverexporter/README.md (hunk 1 above)
Comment (asuresh4) [reply exporter/stackdriverexporter/README.md:1]: Created issues for these

**Review by jrcamp — APPROVED** [review]

**Pull request merged**